release-notes/swan-lake-beta1-release-note.md

```diff
@@ -61,13 +61,56 @@ To view bug fixes, see the [GitHub milestone for Swan Lake Beta1](https://github
 
 ### Standard Library Updates
 
-#### Improvements
+#### New features
 
-##### `graphql` Package
+##### GraphQL Package
+- Add declarative auth configurations
+- Support optional types as inputs
+- Support returning distinct service object unions
+- Support inline fragments
+- Support enums as input values
 
- - Allow enum as an input parameter.
+#### Improvements
 
-#### Bug Fixes
+##### GraphQL Package
+- Improved introspection validation and execution
+- Added missing fields in GraphQL types
+- Compiler plugin improvements to validate inputs and return types
+- Use included record parameters instead of record type in listener initialization
+
+##### HTTP Package
+- Improve `http:Client` remote methods to support contextually expected type inference.
+- Change configuration parameters of listeners and clients to included record parameters.
+
+##### WebSubHub Package
+- Include Auth Configuration to WebSubHub PublisherClient configuration
+
+#### Bug fixes
```

Comment:
```suggestion
#### Bug Fixes
```
We don't mention bug fixes specifically in the release note. We just include the link to the repo filter. Please check other sections on the template.